packages/stack/test-runner/src/lib/index.js

```diff
@@ -175,20 +176,27 @@ class TestRunner {
       }
     };
 
+    global.evmVersion = await this.evmMethod('web3_clientVersion');
```

Comment:
I don't think we should set this to a variable. I see why you did it, so that users can check it before using `evm_time` or whatever, but if they want to use it, they better use `this.evmMethod('web3_clientVersion')` instead or even better, we expose a method that returns the result of `this.evmMethod('web3_clientVersion')`.

The reason for that is that the tests can change client from one test to the other. Eg: the default is the VM, but one test can use the `blockchain` config of the `config()` method and connect to a node temporarily.

Therefore, I think, ourselves, should use the `this.evmMethod('web3_clientVersion')` each time we want to check too, because the original version might not be the real version as of calling the function.

This also fixes your issue of having to call `setupGlobalVariables` after the blockchain is started.